libs/vkd3d-shader/spirv.c

```diff
@@ -9109,6 +9136,12 @@ static void vkd3d_dxbc_compiler_emit_sample(struct vkd3d_dxbc_compiler *compiler
             return;
     }
 
+    if (force_explicit_lod)
+    {
+        operands_mask |= SpvImageOperandsLodMask;
+        image_operands[image_operand_count++] = vkd3d_dxbc_compiler_get_constant_float(compiler, 0.0f);
```

Comment:
is image_operands large enough?

Reply:
Think so, we block Bias and LOD clamp this way.

Reply:
At most we get explicit LOD + texel offset in this path.